private-templates-service/src/config/passport.ts

```diff
@@ -0,0 +1,41 @@
+import passport from 'passport';
+import passportAD from 'passport-azure-ad';
+import config from './config';
+import logger from '../util/logger';
+
+enum LoggingLevels {
```

Comment:
Should probably be in Logger file

Reply:
These are the levels specified by passport library, I added a comment to file.